### DISCUSSION TITLE
How to get stats from the upload command?

### DISCUSSION THREAD

**Discussion by benoitchantre** [deployphp/deployer#2663] [Help needed]
Hello,

Is there a way to use the `upload` command and to display some stats?

I have tried to add the `--stats` option from `rsync`, but nothing is displayed, even when I use `-v` for normal output or `-vv` for more verbose output.

Here's my task reponsible to upload the files from the CI server to the web server.

```php
desc( 'Upload files' );
task( 'deploy:upload', function () {
	run( 'mkdir -p {{deploy_cache}}' );

	upload(
		'release/.',
		'{{deploy_cache}}',
		[
			'flags' => 'rzcE',
			'options' => [
				'--delete',
				'--stats',
			]
		]
	);

	run( 'rsync -rcE --delete {{deploy_cache}}/* {{release_path}}' );
} );
```

Thanks for your help

**Comment by antonmedv** [comment]
Upload func parses rsync output to show progress bar. Use run+rsync.

**Reply by benoitchantre** [reply]
Thank you, I’ll try that way.
